.github/workflows/deploy-portainer.yml

```
@@ -1,539 +1,578 @@
-                  fi
-                done
-              fi
-            fi
-
-            if [ -n "$DEPLOY_PATH" ] && [ -d "$DEPLOY_PATH" ]; then
-              echo "Deploying from host path: $DEPLOY_PATH"
-              deploy_from_host_path "$DEPLOY_PATH"
-              exit 0
-            fi
-
+name: Deploy to Production (SSH)
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)